dist/angular-validation-rule.js

```diff
@@ -14,6 +14,10 @@
         },
         maxlength: function(value, scope, element, attrs, param) {
           return !value || value.length <= param;
+        },
+        equality: function (value, scope, element, attrs, param) {
+          var paramVal = document.querySelector('input[name=' + param + ']').value
```

Comment:
You should be able to use `attrs[ param ]` instead of globally selector from document.